src/util/bip32.h

```diff
@@ -14,6 +14,6 @@
 
 /** Write HD keypaths as strings */
 std::string WriteHDKeypath(const std::vector<uint32_t>& keypath);
-std::string FormatHDKeypath(const std::vector<uint32_t>& path);
+std::string FormatHDKeypath(const std::vector<uint32_t>& path, bool apostrophe = true);
```

Comment:
Could this default to `false` so that we prefer `h`'s everywhere? In my own testing, this only caused a few functional tests to fail, the unit tests seem fine.

Reply:
Done

Reply:
Missed a few spots for legacy, pushed again.